content/client-lib-development-guide/features.textile

```diff
@@ -1336,6 +1338,38 @@ h4. ChannelProperties
 * @(CP2)@ The attributes of @ChannelProperties@ consist of:
 ** @(CP2a)@ @attachSerial@ string - contains the last @channelSerial@ received in an @ATTACHED@ @ProtocolMesage@ for the channel, see "RTL15a":#RTL15a
 
+h4. ChannelDetails
+
+* @(CHD1)@ @ChannelDetails@ is a type that represents information for a channel including channelId, name, status and occupancy
+* @(CHD2)@ The attributes of @ChannelDetails@ consist of:
+** @(CHD2a)@ @channelId@ string - the identifier of the channel
+** @(CHD2b)@ @name@ string - the name of the channel
+** @(CHD2c)@ @status@ @ChannelStatus@ - the status of the channel
+
+h4. ChannelStatus
+
+* @(CHS1)@ @ChannelStatus@ is a type that contains status and occupancy for a channel 
+* @(CHS2)@ The attributes of @ChannelStatus@ consist of:
+** @(CHS2a)@ @isActive@ boolean - represents if the channel is active
+** @(CHS2b)@ @occupancy@ @ChannelOccupancy@ - occupancy is an object containing the metrics for the channel
+
+h4. ChannelOccupancy
+
+* @(CHO1)@ is a type that contain channel metrics
```

Comment:
This line and the one below are missing a capital letter at the start.

Reply:
Oh wait, I can see that this line kind of follows on from the `h4` above... Maybe change it to:
> * @(CHO1)@ A @ChannelOccupancy@ is a type that contain channel metrics

(Inspired by [`TG1`](https://github.com/ably/docs/blob/main/content/client-lib-development-guide/features.textile#paginatedresult)).

The line below should definitely start with a capital though.

Reply:
Done